src/js/themes/base.js

```diff
@@ -278,6 +278,32 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       },
       // extend: undefined,
     },
+    audio: {
+      icons: {
+        pause: Pause,
+        play: Play,
+        volume: Volume,
+        color: 'white',
+      },
+      volume: {
+        background: 'light-4',
+        color: 'brand',
+        opacity: 'strong',
+      },
+      controls: {
+        volume: {
+          rangeInput: {
+            thumb: {
+              color: 'brand',
+            },
+          },
+        },
+        duration: {
```

Comment:
I wonder if `text` would be clearer, since `duration` or `time` feel like setting a time value.